wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md

```diff
@@ -35,15 +38,15 @@ Osu accepts 2 major file formats: mp3 and ogg vorbis. Vorbis has a clear advanta
 Images
 ------
 
-There are 2 major image formats which make up most of the images you find online: jpg and png. Png uses lossless compression which makes it ideal for small and or simple images. Complex images are better suited for jpg. The majority of image editing programs however do not naturally use \*.png and \*.jpg compression to the most of their ability however, leaving you with 4 KB pngs of 1 color, which could be compressed up to 400 times smaller.
+There are 2 major image formats which make up most of the images you find online: jpg and png. Png uses lossless compression which makes it ideal for small and or simple images. Complex images are better suited for jpg. The majority of image editing programs however do not naturally use `.png` and `.jpg` compression to the most of their ability however, leaving you with 4 KB pngs of 1 color, which could be compressed up to 400 times smaller.
 
-[Gimp](http://www.gimp.org/) actually does a good job with \*.jpg files.
+[Gimp](http://www.gimp.org/) actually does a good job with `.jpg` files.
```

Comment:
43: change Gimp to GIMP. GIMP is an acronym that stands for GNU Image Manipulation Program.